meldebot/mel/gif.py

```diff
@@ -44,6 +63,28 @@ def get_gif_url(params):
     return gif_url
 
 
+def get_gif_url_tenor(params):
+    api_key = get_tenor_api_key()
+    if not api_key:
+        logger.critical("NO API KEY FOR Tenor!")
+        exit(-1)
+
+    base_url = "https://api.tenor.com/v1/search?"
+    search_params = "+".join(params)
+    idx = randint(1, 50)
+    query_url = "{url}?q={search}&pos={idx}&limit=1&key={api}&contentfilter=off&media_filter=basic&ar_range=all".format(
+        url=base_url, search=search_params, idx=idx, api=api_key
+    )
+    logger.debug("Tenor - GET: {}".format(query_url))
+    r = http_get(query_url)
```

Comment:
We can open a new issue with this but I think it fits here.
What about making use of requests' escaping methods? 🙄 
You should also fix giphy though...

Something like:
```python
    base_url = "https://api.tenor.com/v1/search"
    search_params = "+".join(params)
    idx = randint(1, 50)
    url_params = {
        'q': search_params,
        'key': api_key,
        'pos': idx,
        'limit': 1,
        'contentfilter': 'off',
        'media_filter': 'basic',
        'ar_range': 'all',
    }
    logger.debug("TENOR - GET: {}".format(url_params ))
    r = http_get(url=base_url, params=url_params)
```